File: lib/ts/src/internal/schema/GetInternalTelepactJson.ts
//|
//|  Copyright The Telepact Authors
//|  SPDX-License-Identifier: Apache-2.0
//|

import internalTelepact from '../../../inc/internal.telepact.json';

export function getInternalTelepactJson(): string {
    return JSON.stringify(internalTelepact);
}

File: lib/go/internal/schema/GetInternalTelepactJSON.go
//|
//|  Copyright The Telepact Authors
//|  SPDX-License-Identifier: Apache-2.0
//|

package schema

import "sync"

var (
	internalTelepactJSON     string
	internalTelepactJSONOnce sync.Once
)

// GetInternalTelepactJSON returns the bundled Telepact internal schema JSON content.
func GetInternalTelepactJSON() string {
	internalTelepactJSONOnce.Do(func() {
		content, err := loadBundledSchema("internal.telepact.json")
		if err != nil {
			panic(err)
		}
		internalTelepactJSON = content
	})
	return internalTelepactJSON
}
